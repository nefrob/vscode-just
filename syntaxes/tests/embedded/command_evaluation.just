# command evaluation

localhost := `dumpinterfaces | cut -d: -f2 | sed 's/\/.*//' | sed 's/ //g'`

stuff := ```
    echo foo
    echo bar
  ```