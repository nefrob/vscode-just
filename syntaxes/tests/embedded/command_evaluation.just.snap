># command evaluation
#^^^^^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>localhost := `dumpinterfaces | cut -d: -f2 | sed 's/\/.*//' | sed 's/ //g'`
#^^^^^^^^^ source.just variable.other.just
#         ^ source.just
#          ^^ source.just keyword.operator.assignment.just
#            ^ source.just
#             ^ source.just string.interpolated.just
#              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.just source.shell
#                                                                          ^ source.just string.interpolated.just
>
>stuff := ```
#^^^^^ source.just variable.other.just
#     ^ source.just
#      ^^ source.just keyword.operator.assignment.just
#        ^ source.just
#         ^ source.just string.interpolated.just
#          ^ source.just string.interpolated.just
#           ^ source.just
>    echo foo
#^^^^^^^^^^^^^ source.just
>    echo bar
#^^^^^^^^^^^^^ source.just
>  ```
#^^ source.just
#  ^ source.just string.interpolated.just
#   ^ source.just string.interpolated.just
#    ^^ source.just